tests/assets/eks_node_role_policy.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
I don't see this file being used anywhere. Did I miss anything?

Reply:
removed it as it is not getting used.